compiler/src/dotty/tools/backend/jvm/BytecodeWriters.scala

```diff
@@ -117,7 +117,8 @@ trait BytecodeWriters {
       catch case ex: ClosedByInterruptException =>
         try
           outfile.delete() // don't leave an empty or half-written classfile around after an interrupt
-        catch case _: Throwable =>
+        catch
+          case _: Throwable =>
```

Comment:
```suggestion
        catch case ioe: Throwable => ex.addSuppressed(ioe)
```

Reply:
I just want to fix the issue here, not do style improvements in code of which I know nothing.